app/src/generate.py

```diff
@@ -1,23 +1,20 @@
 import logging
 import os
-from typing import Any, Sequence
+from typing import Any
 
 from litellm import completion
 
-from src.citations import create_prompt_context
-from src.db.models.document import Chunk
-
 logger = logging.getLogger(__name__)
 
 PROMPT = """Provide answers in plain language written at the average American reading level.
 Use bullet points. Keep your answers brief, max of 5 sentences.
 Keep your answers as similar to your knowledge text as you can
 
 When referencing the context, do not quote directly.
-Use the provided citation numbers (e.g., (citation-0)) to indicate when you are drawing from the context.
-To cite multiple sources at once, you can append citations like so: (citation-0)(citation-1), etc.
+Use the provided citation numbers (e.g., (citation-1)) to indicate when you are drawing from the context.
+To cite multiple sources at once, you can append citations like so: (citation-1)(citation-2), etc.
 Place the citations after any closing punctuation for the sentence.
-For example: 'This is a sentence that draws on information from the context.(citation-0)'
+For example: 'This is a sentence that draws on information from the context.(citation-1)'
```

Comment:
LLM generated response that cited nonexistent `(citation-0)`. This change fixed that issue.
See https://nava.slack.com/archives/C06DP498D1D/p1728502233060729

Reply:
If this problem recurs, we could create another `CitationIdFactory` that provides random citation ids to maybe prevent hallucinations.